bin/pt-online-schema-change

```diff
@@ -9181,6 +9181,42 @@ sub main {
          channel      => $o->get('channel'),
       );
 
+      # Check if we are not a replica of the source server with ROW or MIXED base replication
+      if ( !$o->get('force') ) {
+         my $source = $ms->get_source_dsn($cxn->dbh(), $dsn, $dp);
+         if ( $source ) {
+            my $source_cxn = $make_cxn->(dsn => $source);
+
+            # Check source
+            my $is_source_of = eval {
+               $ms->is_source_of($source_cxn->{dbh}, $cxn->{dbh});
+            };
+
+            # We should not die if replica connected via tunnel or port redirection
+            if ( $EVAL_ERROR ) {
+               $EVAL_ERROR =~ m/The replica is connected to (\d+) but the source's port is \d+/;
+               if ( !$1 || $1 != $source->{P} ) {
+                  $is_source_of = 0;
+               }
+            }
+
+            if ( $is_source_of) {
+               my $source_binlog_format = $source_cxn->dbh()->selectrow_arrayref("SHOW GLOBAL VARIABLES LIKE 'binlog_format'");
+               use Data::Dumper;
```

Comment:
we have this use at the beginning of the script but we load only qw(). So better to update it at the beginning

Reply:
Oh, this `use Data::Dumper` should not be here at all, removed.